Pipfile

```diff
@@ -19,29 +23,25 @@ decorator = "*"
 demisto-py = ">=3.1.1"
 dictor = "*"
 docker = "==4.3.0"
-flake8 = "*"
 flatten-dict = "*"
 gitdb = "*"
 google-cloud-storage = "==1.29.0"
 inflection = "==0.5.1"
 isort = "*"
 jsonschema = "*"
 mergedeep = "*"
-mypy = "==0.782"
-networkx = "*"
+networkx = "==2.6.3"
```

Comment:
please add a comment explaining why we lock this version

Reply:
thanks. We'll unlock the version as part of [this issue](https://github.com/demisto/etc/issues/47123).